### PR TITLE
5 1j loginregistration page

### LINKED ISSUE #5: 1J - Login/Registration page

### PR DESCRIPTION
**Pull request by larroyo3**
Closes #5 
--

Create the logInSignInActivity with 
- btn logIn
- btn SignIn
- link to privacy policy et condition d'utilisation

> And more
Fix conflit with last PR and connected registrationActivty and LogInSignInMenu

### CODE REVIEW

File: app/src/androidTest/java/com/bouboulecorp/thurii/ExampleInstrumentedTest.kt [hunk 1]
```diff
@@ -1,12 +1,19 @@
 package com.bouboulecorp.thurii
 
+import androidx.test.espresso.Espresso.onView
```
Comment (Neyross) [inline app/src/androidTest/java/com/bouboulecorp/thurii/ExampleInstrumentedTest.kt:3]: 🚫 **[ktlint]** <import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between

Comment (Neyross) [inline app/src/androidTest/java/com/bouboulecorp/thurii/ExampleInstrumentedTest.kt:3]: 🚫 **[ktlint]** <no-unused-imports> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unused import

File: app/src/androidTest/java/com/bouboulecorp/thurii/ExampleInstrumentedTest.kt [hunk 2]
```diff
@@ -1,12 +1,19 @@
...
+import androidx.test.espresso.action.ViewActions.click
```
Comment (Neyross) [inline app/src/androidTest/java/com/bouboulecorp/thurii/ExampleInstrumentedTest.kt:4]: 🚫 **[ktlint]** <no-unused-imports> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unused import

File: app/src/androidTest/java/com/bouboulecorp/thurii/ExampleInstrumentedTest.kt [hunk 3]
```diff
@@ -1,12 +1,19 @@
...
+import androidx.test.espresso.assertion.ViewAssertions.matches
```
Comment (Neyross) [inline app/src/androidTest/java/com/bouboulecorp/thurii/ExampleInstrumentedTest.kt:5]: 🚫 **[ktlint]** <no-unused-imports> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unused import

File: app/src/androidTest/java/com/bouboulecorp/thurii/ExampleInstrumentedTest.kt [hunk 4]
```diff
@@ -1,12 +1,19 @@
...
+import androidx.test.espresso.matcher.ViewMatchers.*
```
Comment (Neyross) [inline app/src/androidTest/java/com/bouboulecorp/thurii/ExampleInstrumentedTest.kt:6]: 🚫 **[ktlint]** <no-wildcard-imports> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wildcard import (cannot be auto-corrected)

File: app/src/androidTest/java/com/bouboulecorp/thurii/ExampleInstrumentedTest.kt [hunk 5]
```diff
@@ -1,12 +1,19 @@
...
+import androidx.test.ext.junit.rules.ActivityScenarioRule
```
Comment (Neyross) [inline app/src/androidTest/java/com/bouboulecorp/thurii/ExampleInstrumentedTest.kt:7]: 🚫 **[ktlint]** <no-unused-imports> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unused import

File: app/src/androidTest/java/com/bouboulecorp/thurii/ExampleInstrumentedTest.kt [hunk 6]
```diff
@@ -1,12 +1,19 @@
...
 import androidx.test.platform.app.InstrumentationRegistry
 import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.filters.LargeTest
```
Comment (Neyross) [inline app/src/androidTest/java/com/bouboulecorp/thurii/ExampleInstrumentedTest.kt:10]: 🚫 **[ktlint]** <no-unused-imports> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unused import

File: app/src/androidTest/java/com/bouboulecorp/thurii/ExampleInstrumentedTest.kt [hunk 7]
```diff
@@ -1,12 +1,19 @@
...
 
 import org.junit.Test
 import org.junit.runner.RunWith
 
 import org.junit.Assert.*
+import org.junit.Rule
```
Comment (Neyross) [inline app/src/androidTest/java/com/bouboulecorp/thurii/ExampleInstrumentedTest.kt:16]: 🚫 **[ktlint]** <no-unused-imports> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unused import

File: app/src/androidTest/java/com/bouboulecorp/thurii/LogInSignInMenu/LogInSignInMenuTests.kt [hunk 8]
```diff
@@ -0,0 +1,37 @@
+package com.bouboulecorp.thurii.LogInSignInMenu
```
Comment (Neyross) [inline app/src/androidTest/java/com/bouboulecorp/thurii/LogInSignInMenu/LogInSignInMenuTests.kt:1]: 🚫 **[ktlint]** <final-newline> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>File must end with a newline (\n)

File: app/src/androidTest/java/com/bouboulecorp/thurii/LogInSignInMenu/LogInSignInMenuTests.kt [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+
+import androidx.test.core.app.ActivityScenario
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
```
Comment (Neyross) [inline app/src/androidTest/java/com/bouboulecorp/thurii/LogInSignInMenu/LogInSignInMenuTests.kt:5]: 🚫 **[ktlint]** <no-unused-imports> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unused import

File: app/src/androidTest/java/com/bouboulecorp/thurii/LogInSignInMenu/LogInSignInMenuTests.kt [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.internal.runner.junit4.AndroidJUnit4ClassRunner
+import com.bouboulecorp.thurii.LogIn.LoginActivity
```
Comment (Neyross) [inline app/src/androidTest/java/com/bouboulecorp/thurii/LogInSignInMenu/LogInSignInMenuTests.kt:11]: 🚫 **[ktlint]** <no-unused-imports> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unused import

File: app/src/androidTest/java/com/bouboulecorp/thurii/RegistrationTests/FragmentRegistrationGenderTest.kt [hunk 11]
```diff
@@ -8,9 +8,11 @@ import androidx.test.espresso.matcher.ViewMatchers.withText
 import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
 import androidx.test.espresso.matcher.ViewMatchers.withId
 import androidx.test.internal.runner.junit4.AndroidJUnit4ClassRunner
+import com.bouboulecorp.thurii.LogInSignInMenu.LogInSignInMenuActivity
```
Comment (Neyross) [inline app/src/androidTest/java/com/bouboulecorp/thurii/RegistrationTests/FragmentRegistrationGenderTest.kt:11]: 🚫 **[ktlint]** <no-unused-imports> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unused import

File: app/src/main/java/com/bouboulecorp/thurii/LogInSignInMenu/LogInSignInMenuActivity.kt [hunk 12]
```diff
@@ -0,0 +1,36 @@
+package com.bouboulecorp.thurii.LogInSignInMenu
+import android.content.Intent
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/LogInSignInMenu/LogInSignInMenuActivity.kt:2]: 🚫 **[ktlint]** <import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between

File: app/src/androidTest/java/com/bouboulecorp/thurii/RegistrationTests/FragmentRegistrationGenderTest.kt [hunk 13]
```diff
@@ -8,9 +8,11 @@ import androidx.test.espresso.matcher.ViewMatchers.withText
...
 import com.bouboulecorp.thurii.MainActivity
```
Comment (Neyross) [inline app/src/androidTest/java/com/bouboulecorp/thurii/RegistrationTests/FragmentRegistrationGenderTest.kt]: 🚫 **[ktlint]** <no-unused-imports> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unused import

**Review by claraxuxu — COMMENTED** [review]
Good job

**Pull request merged**